### PR TITLE
Clean up some unnecessary emits to ProjectFacts

### PR DESCRIPTION
**Pull request by thelee**

### CODE REVIEW

File: relational/table.py [hunk 1]
```diff
@@ -318,22 +318,33 @@ def _gen_units(self, row, proj):
             row[self.index(self.NET_NUM_UNITS_BMR_DATA)] = source
         else:
             dbi_net = _get_dbi_units(proj)
+            ppts_net = proj.field('market_rate_units_net', PPTS.NAME)
```
Comment (rajpara11) [inline relational/table.py]: Hmm unrelated, but I thought we had said we wanted to use the total project units here and not just the market rate units?

File: relational/table.py [hunk 2]
```diff
@@ -318,22 +318,33 @@ def _gen_units(self, row, proj):
...
             net = dbi_net
-            if dbi_net is not None:
+            # PTS may have an explicitly set 0 unit count for projects
+            # that have no business dealing with housing (possible with
+            # permit type 3 in particular), so we only emit a 0-count
+            # PTS unit count if we had an explicit non-0 PPTS unit
+            # count (therefore indicating a housing-related project that
```
Comment (rajpara11) [inline relational/table.py]: Curious more than anything else, are these 'rehab' projects that show up like this typically?

File: relational/table.py (hunk 1 above)
Comment (thelee) [reply relational/table.py]: There's issue #48: iiuc this is just a naming issue in the schema-less

Comment (rajpara11) [reply relational/table.py]: sg

File: relational/table.py (hunk 2 above)
Comment (thelee) [reply relational/table.py]: i only looked at a few, but they were like very minor things. they had very little other information in the other tables, so they were basically just noisy zeros.

Comment (rajpara11) [reply relational/table.py]: sg, probably the best approach that we can take here in that case

**Pull request merged**